src/main/kotlin/dev/schlaubi/votebot/commands/ClaimPermissionsCommand.kt

```diff
@@ -35,7 +36,7 @@ object ClaimPermissionsCommand : SingleCommand() {
 
     override suspend fun execute(context: Context) {
         if (Permission.Administrator !in context.executor.asMember().getPermissions()) {
-            context.respond("You need the `ADMINISTRATOR` permission to execute this command")
+            context.respond(Embeds.error("You need the `ADMINISTRATOR` permission to execute this command", description = null))
```

Comment:
The description is null per default

Reply:
yeah, forgot to remove it again since at the beginning it was not null by default